ourjapanlife/findadoc-server
# feat: add PhysicalAddress to Contact

Part of #87


# What changed
1. Adds the `PhysicalAddress` type to `Facility`
2. Creates a migration to add the PhysicalAddress columns
3. Updates the seeds to include the Ja fields for PhysicalAddress
4. Adds query and resolver functions for getting a PhysicalAddress by Id or a list of PhysicalAddresses

# Testing instructions
1. Run `docker compose up`
6. Run `yarn dev`
7. Use the following queries in Apollo server: 
(See "Testing GraphQL" section in README for more detailed steps)

## Test 1: Query PhysicalAddress in Facilities
Operations:
```
query Query {
  facilities {
    physicalAddress {
      postalCode
      addressLine1Ja
      addressLine1En
    }
  }
}
```
## Test 2: Query all PhysicalAddresses
Operations:
```
query Query {
  physicalAddresses {
    addressLine1En
    addressLine1Ja
  }
}
```

## Test 3: Query PhysicalAddress by Id
Operations:
```
query Query($physicalAddressId: ID!) {
  physicalAddress(id: $physicalAddressId) {
    id
    addressLine1En
    addressLine1Ja
  }
}
```

Variables:
```
{
  "physicalAddressId": "2"
}
```
